packages/core/src/components/portal/portal.tsx

```diff
@@ -68,7 +76,7 @@ export class Portal extends React.Component<IPortalProps, IPortalState> {
 
     public componentDidMount() {
         this.portalElement = this.createContainerElement();
-        document.body.appendChild(this.portalElement);
+        this.props.container.appendChild(this.portalElement);
```

Comment:
check that this exists first

Reply:
Resolved by https://github.com/palantir/blueprint/pull/3045/commits/de0049868797e1c38b7a415136022dd6febf3024